scene/2d/tile_map.h

```diff
@@ -277,6 +278,9 @@ class TileMap : public Node2D {
 
 	void update_dirty_quadrants();
 
+	void set_show_collision(bool p_value);
+	bool is_show_collision_enabled();
```

Comment:
Just a detail, but this could be `bool is_show_collision_enabled() const;`. I didn't see it before, sorry :)